### PR TITLE
Feature/nfwmc ellipse

### PR DESCRIPTION
**Pull request by ajshajib**
This PR adds an elliptical NFW mass model parametrized in the mass-concentration space, extending the previously available class for the spherical case.

### CODE REVIEW

File: lenstronomy/LensModel/Profiles/nfw_mass_concentration_ellipse.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    def __init__(self, z_lens, z_source, cosmo=None, static=False):
+        """
+
+        :param z_lens: redshift of lens
+        :param z_source: redshift of source
+        :param cosmo: astropy cosmology instance
+        :param static: boolean, if True, only operates with fixed parameter values
+        """
+        super(NFWMCEllipse, self).__init__(z_lens, z_source, cosmo, static)
+        self._nfw = NFW_ELLIPSE()
```
Comment (sibirrer) [inline lenstronomy/LensModel/Profiles/nfw_mass_concentration_ellipse.py:50]: I tend to transition to the elliptical convergence NFW profile as the new default 'CSE', which is accurate and more physical, I am also thinking to change the convention overall of NFW_ELLIPSE to query the CSE decomposed profile

**Review by sibirrer — COMMENTED** [review]
Thank you very much @ajshajib !
Looks good to me. I have one comment since you use an ellipse whether we should move it to use the ellipse defined in convergence, which is much more physical?

File: lenstronomy/LensModel/Profiles/nfw_mass_concentration_ellipse.py (hunk 1 above)
Comment (ajshajib) [reply lenstronomy/LensModel/Profiles/nfw_mass_concentration_ellipse.py]: That sounds good to me. It's done. 🙂

**Review by sibirrer — APPROVED** [review]
looks great, thank you!

**Pull request merged**